### DISCUSSION TITLE
Access NormCounts data used by plotTrajectory function

### DISCUSSION THREAD

**Discussion by jblich870** [GreenleafLab/ArchR#1373] [Questions / Documentation] [answered]
Hi,

I wondered how to access the 'NormCounts' data that is used in the plotTrajectory function?

It does not appear to be identical to the data stored in "GeneExpressionMatrix".

Many thanks,
J

**Comment by rcorces** [comment, marked as answer]
`plotTrajectory()` obtains the values from the specified matrix here:
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/Trajectory.R#L778

If you follow what happens to `plotParams$color`, The additional manipulations after getting these matrix values include:
1. applying `imputeWeights` which will happen automatically if they are available and you havent set `imputeWeights = NULL`. https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/Trajectory.R#L824
2. quantile cut which will happen when plotting things like gene expression. https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/Trajectory.R#L831

If you wanted to obtain these values, you should be able to get them from the plot object for the particular gene you are plotting or perform the manipulations yourself after looking at the code links above.

**Reply by jblich870** [reply]
Brilliant, really helpful - thank you!
